src/simulation/LoadGenerator.cpp

```diff
@@ -477,17 +510,55 @@ LoadGenerator::findAccount(uint64_t accountId, uint32_t ledgerNum)
 
 LoadGenerator::TxInfo
 LoadGenerator::paymentTransaction(uint32_t numAccounts, uint32_t offset,
```

Comment:
I left it as Payment since the LoadGen mode is `pay`. Let me know what you think!

Reply:
As @marta-lokhova pointed out (and I agree), it's fairly confusing to create `SetOptions` in the `pay` mode. The config contains magic numbers, the `pay` mode may or may not produce `PaymentOps`. 

I'm thinking that I'd update this PR to something like
- Keep `create` and `pay` as is.
- Add a new mode `pubnet-txn` and this mode creates transactions containing `SetOptions`.
	- I'm sure there's a better name and I'm definitely open to suggestions!
	- I picked it since what we really want is to send transactions that are more similar to transactions in pubnet in some way. 

I think this is a good design since...
- Updating `a` and `b` in the future would be much easier. 
- The interface reflects the use case better. I imagine that users of this feature wouldn't care if it's 48, 52 or 56 bytes.  They'd care if it's similar to what's observed in the pubnet.

What do you think? @marta-lokhova @MonsieurNicolas

Reply:
that's fine with me, like I said before I don't know if we should mention pubnet in the name, but we can call the mode `simulate` or something like that.

Reply:
I updated it the code by adding a new mode `pretend`:

- no change to `create` 
- `pay` is almost the same, but now one can create transactions containing multiple payment ops
- `pretend` is the new mode that helps generate transactions of somewhat realistic sizes to "pretend" as if they have real traffic.